src/wf_utils/labels_tw.config

```
@@ -34,6 +34,14 @@ process {
   withLabel: highsharedmem {
     containerOptions = { workflow.containerEngine != 'singularity' ? "--shm-size ${String.format("%.0f",task.memory.mega * 0.25)}" : ""}
   }
+  withLabel: gpu {
+    //cpus = 31 // misschien dit eerst uitzetten?
+    //memory = 110.GB // misschien dit eerst uitzetten?
+    accelerator = 1
+    // queue = ... // todo: might need to select a gpu specific queue
+    containerOptions = { workflow.containerEngine == "singularity" ? '--nv':
+       ( workflow.containerEngine == "docker" ? '--gpus all': null ) }
+  }
```

Remove comments :sweat_smile: 

```suggestion
  withLabel: gpu {
    accelerator = 1
    containerOptions = { workflow.containerEngine == "singularity" ? '--nv':
       ( workflow.containerEngine == "docker" ? '--gpus all': null ) }
  }
```